src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
if this file what is does is simply importing routes i think you can merge it with indes.js a